internal/gengapic/testdata/foo_rest_client_init.want

```diff
@@ -83,15 +83,6 @@ func (c *fooRESTClient) Zip(ctx context.Context, req *mypackagepb.Bar, opts ...g
 	}
 	return nil, nil
 
-}
```

Comment:
Should there be a method to replace this?

Reply:
No, b.c the REST transport hasn't implemented Paginated methods yet. It was generated b.c it was incorrectly being viewed as a standard Unary RPC (which REST transport recently implemented)